clients/include/testing_matmul.hpp

```diff
@@ -1536,20 +1536,20 @@ void testing_matmul_with_bias(const Arguments& arg,
                                                        B_row[i],
                                                        B_col[i],
                                                        stream));
-                CHECK_HIP_ERROR(synchronize(hScaleB[i], dScaleB[i]));
+                CHECK_HIP_ERROR(transfer_from(hScaleB[i], dScaleB[i]));
```

Comment:
why about here?

Reply:
the function name has been set back to the original.